### PR TITLE
Migrate Snowflake driver test to GitHub Actions

### PR DESCRIPTION
**Pull request by nemanjaglumac**
The gotcha was to paste the private key as one-liner.

```
----BEGIN PRIVATE KEY----- THIS_WORKS  -----END PRIVATE KEY-----
```

```
----BEGIN PRIVATE KEY-----
THIS_DOESN'T
-----END PRIVATE KEY-----
```

As I was typing this I realized that when I formatted the PK with the newline, I probably omitted whitespace.

### CODE REVIEW

File: .github/workflows/drivers.yml [hunk 1]
```diff
@@ -346,3 +346,22 @@ jobs:
       uses: ./.github/actions/test-driver
       with:
         junit-name: 'be-tests-sqlserver-ee'
+
+  be-tests-snowflake-ee:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 120
```
Comment (dpsutton) [inline .github/workflows/drivers.yml:352]: can you make a follow up issue that in a month we look at the historical duration of this new test? I'd like to bump this down as soon as we have a baseline for how long it should run. 2 hours seems wild. If in a month we find these tests do take 2 hours, we should fix that for sure.

**Review by dpsutton — APPROVED** [review]

File: .github/workflows/drivers.yml (hunk 1 above)
Comment (flamber) [reply .github/workflows/drivers.yml:352]: Likely because of issue 26054, so it shouldn't need special timeout from now on.

Comment (nemanjaglumac) [reply .github/workflows/drivers.yml:352]: I think we fixed problems with Snowflake that caused it to run that long. At one point we had to bump the timeout in CircleCI to 115 minutes.

We'll closely monitor all driver tests. As @camsaul noted in another PR, all driver tests seem to be getting slower and slower. Maybe we should investigate that once we migrate them all to GitHub Actions.

Comment (nemanjaglumac) [reply .github/workflows/drivers.yml]: Actually, I had to rebase to solve the merge conflicts anyway so I dialed down this to the 60 mintues. cc @dpsutton

**Pull request merged**